clang/test/CAS/cas-backend.c

```diff
@@ -1,8 +1,8 @@
-// RUN: rm -rf %t && mkdir -p %t
-// RUN: llvm-cas --cas %t/cas --ingest --data %s > %t/casid
+// RUN: rm -rf %t %t.cas && mkdir -p %t
+// RUN: llvm-cas --cas %t.cas --ingest --data %s > %t/casid
```

Comment:
This test doesn't need change, it's ingesting `%s`, was changing it intentional for consistency? There are some other tests like this as well.

Reply:
Oh, I saw so many failures inclang/test/CAS that I assumed every test that used llvm-cas --ingest was failing, so I just changed them all. I can change these two back if you like, though it seems harmless.

Reply:
To be precise, half of these tests do not need changing, but it's up to you, I don't particularly care either way.

Reply:
Hmm, you're right. Not sure what I was seeing before. Reverted the unnecessary changes.